R/method-correlation.R

```diff
@@ -163,6 +170,9 @@ buildCorrelationComputeResult <- function(corrResult, data1, data2 = NULL, metho
 #' @return a ComputeResult object
 #' @export
 setMethod("correlation", signature("AbundanceData", "missing"), function(data1, data2, method = c('spearman','pearson'), verbose = c(TRUE, FALSE)) {
+  #prefilters applied
+  data1 <- pruneFeatures(data1, halfOfSamplesHaveNonZeroAbundance, verbose)
```

Comment:
If we think we're going to let the user make some of these pruning decisions in the future, why not add arguments to our correlation functions and just set these as defaults?

Reply:
it probably makes sense.. i expect were going to iterate here though, and if im going to be revisiting this anyway id rather make api changes once i know more about our needs.

Reply:
... i guess the one about proportion of samples w non-zero values will probably def be around. maybe i can work that one in at least.

Reply:
ive had a better idea.. hold please